Monika After Story/game/definitions.rpy

```diff
@@ -1273,6 +1273,14 @@ python early:
                 # TODO: same TODO as in checkConditionals.
                 #   indexing would be smarter.
 
+                #If this is on load and the event action is a push/queue without post greet, we skip its eval here
+                if (
+                    on_load
+                    and ev.action in [EV_ACT_PUSH, EV_ACT_QUEUE]
+                    and "postgreet" not in ev.rules
+                ):
+                    continue
```

Comment:
`continue` leads to bad code. if this just needs to return early then have it return.

Reply:
This doesn't need to return. The rest of the events should still be processed if need be

Reply:
then change the condition to be reversed and put the event check inside it. `continue` is sloppy.